docs/docs/guide/debugging.mdx

```diff
@@ -0,0 +1,200 @@
+---
+id: debugging
+title: "Debugging Reanimated"
+sidebar_label: "Debugging Reanimated"
+---
+
+import {
+  SummaryTable,
+  ChromeDebuggerTable,
+  ChromeDevToolsTable,
+  FlipperTable,
+  SafariDevToolsTable,
+  ReactDeveloperToolsTable,
+} from './debugging_tables/tables';
+
+Due to Reanimated's unique architecture and usage of a second thread debugging
+can be problematic and some common solutions might not work as expected.
+This article summarizes all the available methods and highlights their caveats.
+
+:::info
+
+Reanimated v1 works with all the common React Native debugging tools. This
+article focuses on the use of these tools with Reaniamted v2.
+
+:::
+
+### React Native debugging tools
+
+These are the debugging tools we checked for compatibility with React Native
+apps using the Reanimated library:
+
+- [**Chrome debugger**](https://reactnative.dev/docs/debugging#chrome-developer-tools)
+(also known as the _React Native Debugger_) is a simple tool that creates a web
+worker inside your preferred browser which executes your app's JavaScript code. It works
+with all the runtimes available in React Native.
+
+- [**Chrome DevTools**](https://reactnative.dev/docs/hermes#debugging-js-on-hermes-using-google-chromes-devtools)
+(unavailable for JSC) is Chrome's built-in feature that allows for connecting to a
+remote JavaScript runtime. This means that the code you are debugging actually
+runs on the device itself and behaves the same way as it would with DevTools detached.
+
+- [**Flipper**](https://fbflipper.com/) (Hermes debugger) is an app created by
+Facebook that makes it easy to use the _Chrome DevTools_ while providing additional tools for
+UI inspection and debugging.
+
+- [**Safari DevTools**](https://reactnative.dev/docs/debugging#safari-developer-tools)
+is Safari's built-in feature that is only available for iOS devices running JSC. It works
+very similarly to Chrome DevTools - it allows you to connect to a remote runtime
+and keep code execution on device.
+
+- [**React Developer Tools**](https://reactnative.dev/docs/debugging#react-developer-tools)
+are a standalone app that allows debugging UI through the inspector as well as monitoring
+performance and profiling your app.
+
+### Debugging web apps
+
+For debugging web apps you may use the tools you prefer or any that are provided
+by the browser of you choice. We did not discover any issues with debugging
+apps using Reanimated on web.
+
+### Summary
+
+<SummaryTable />
+
+### Chrome Debugger
+
+<ChromeDebuggerTable />
+
+:::info
```

Comment:
I feel like we shouldn't use info boxes for TL;DRs of these sections. Can we convert this to a regular paragraph? If you want to emphasize this summary, we can use `fontWeight: bold` or add "**Summary:**" (or similar) at the beginning.